recipe/meta.yaml

```diff
@@ -18,34 +17,48 @@ requirements:
   host:
     - python
     - pip
-    - hatchling
     - wheel
   run:
     - python
-    - importlib-metadata  # [py<38]
     - jinja2
-    - jsonschema >=3.0
+    - jsonschema >=3.0,<4.17
```

Comment:
jsonschema not upper boundary ?
https://github.com/altair-viz/altair/blob/3b58c2adf7a0ce8ce2e25cb21c6b56b6de001526/requirements.txt#L3